tools/db_stress.cc

```diff
@@ -1338,20 +1340,25 @@ class StressTest {
  private:
   Status AssertSame(DB* db, ColumnFamilyHandle* cf,
                     ThreadState::SnapshotState& snap_state) {
+    Status s;
+    if (cf->GetName() != snap_state.cf_at_name) {
```

Comment:
you might want to add `thread->shared->LockColumnFamily(cf);` to prevent race condition in name changing, though it looks extremely unlikely to happen so I'll accept anyways.

Reply:
I was about to add it but it seems that we are already protected by GetMutexForKey a few lines above, since LockColumnFamily would lock all the keys in the cf and as long as one key is locked the cf is not going to be changed.